spec/hallmonitor/outputters/dogstatsd_outputter_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+require 'hallmonitor/outputters/dogstatsd_outputter'
+
+module Hallmonitor
+  module Outputters
+    RSpec.describe DogstatsdOutputter do
+      let(:dogstatsd_client) { instance_double(Datadog::Statsd) }
+      let(:prefix) { 'test' }
+      let(:outputter) { described_class.new(prefix) }
+
+      before do
+        allow(dogstatsd_client).to receive(:namespace=)
+        allow(Datadog::Statsd).to receive(:new).and_return(dogstatsd_client)
+      end
+
+      it 'can be instantiated' do
+        expect { outputter }.to_not raise_error
+      end
+
+      context '#process' do
+        let(:event_name) { 'foo.bar.baz' }
+        let(:event_tags) { {tags: [{foo:"bar"}]} }
+        let(:event_tags_expected) {{tags:["tags:[{:foo=>\"bar\"}]"]}}
```

Comment:
This doesn't look like it's right.  The code that this tests is converting the tags hash into an array that looks like: `['key:value', 'key:value']`, so i would expect this to maybe look like this:
`let(:event_tags_expected) { {tags: ['foo:bar']} }`